tools/pipelines/test-real-service.yml

```
@@ -80,16 +80,35 @@ stages:
           displayName: '[end-to-end tests] npm run test:realsvc:routerlicious:report'
           continueOnError: true
           env:
-            fluid__webpack__bearerSecret: $(fluid-webpack-bearerSecret)
-            fluid__webpack__fluidHost: $(fluid-webpack-fluidHost)
-            fluid__webpack__tenantId: $(fluid-webpack-tenantId)
-            fluid__webpack__tenantSecret: $(fluid-webpack-tenantSecret)
+            fluid__test__driver__r11s: $(automation-fluid-test-driver-r11s)
             FLUID_TEST_LOGGER_PKG_PATH: ${{ variables.testWorkspace }}/node_modules/@ff-internal/aria-logger # Contains getTestLogger impl to inject
           inputs:
             command: 'custom'
             workingDir: ${{ variables.testWorkspace }}/node_modules/@fluidframework/test-end-to-end-tests
             customCommand: 'run test:realsvc:routerlicious:report'
 
+  # end-to-end tests routerlicious
+  - stage:
+    displayName: e2e - routerlicious
```

```suggestion
    displayName: e2e - frs
```
or anything else to differentiate